### DISCUSSION TITLE
Animal money reward

### DISCUSSION THREAD

**Discussion by Khhunt** [janstehno/cotw-companion#16] [Ideas]
How much does a animal give you money after you harvest it. Like 800-1200£

**Comment by janstehno** [comment]
This could be possible. However killing each animal under different conditions to determine the best and worst outcome will take a lot of time.

**Comment by janstehno** [comment]
So I did some research for this idea. Based on my recorded 150 harvests under various conditions, this probably won't be possible. The differences between cash rewards based on difficulty, trophy score, weight, quick kill bonus, etc. are so huge I won't be able to do it for each animal. The developers probably have some sort of formula the game is calculating the cash reward by. And I have no way of knowing it. I could approximate it, but I would need much more harvests than 150.

Unless there is some different idea how I could approach this, I will not implement it.
